integrations/docker/run.sh

```diff
@@ -101,4 +101,6 @@ for arg in "$@"; do
     esac
 done
 
-docker run "${runargs[@]}" --rm --mount "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind" -w "$RUN_DIR" -v "$RUN_DIR:$RUN_DIR" "$ORG/$IMAGE${VERSION:+:${VERSION}}" "$@"
+docker pull "$ORG/$IMAGE${VERSION:+:${VERSION}}" || docker build -t "$IMAGE" "$here" --build-arg VERSION="$VERSION"
+
+docker run "${runargs[@]}" --rm --mount "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind" -w "$RUN_DIR" -v "$RUN_DIR:$RUN_DIR" "$IMAGE" "$@"
```

Comment:
* why are you dropping the $ORG and $VERSION from the run command?
* why aren't you using the local "build.sh" script?